akka-stream-tests/src/test/java/akka/stream/stage/JavaIdentityStage.java

```diff
@@ -0,0 +1,46 @@
+package akka.stream.stage;
+
+import akka.stream.Attributes;
+import akka.stream.FlowShape;
+import akka.stream.Inlet;
+import akka.stream.Outlet;
+
+public class JavaIdentityStage<T> extends GraphStage<FlowShape<T, T>> {
+  private Inlet<T> _in = Inlet.of("Identity.in");
```

Comment:
Well, it is also quite weird to have `create` and `of` depending which sounds better in a certain context. Let's have one, consistent Java creator name then.